<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDialog</class>
 <widget class="QDialog" name="EditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icon/image1/copy-double.png</normaloff>:/icon/image1/copy-double.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>341</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>当前记录</string>
   </property>
   <widget class="QSpinBox" name="spinBox_id">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>31</y>
      <width>248</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>58</y>
      <width>248</width>
      <height>19</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_sex">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>84</y>
      <width>248</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_birthdata">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>111</y>
      <width>248</width>
      <height>19</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_part">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>164</y>
      <width>248</width>
      <height>19</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_money">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>190</y>
      <width>248</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_part">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>137</y>
      <width>248</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>165</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>部门</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>138</y>
      <width>60</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>出生地</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>112</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>出生日期</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>85</y>
      <width>60</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>性别</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>59</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>7</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>32</y>
      <width>60</width>
      <height>18</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>工号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>工资</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_submit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>351</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>351</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>EditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
